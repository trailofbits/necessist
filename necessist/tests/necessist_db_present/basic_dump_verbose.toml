args = ["--root=fixtures/basic", "--dump", "--verbose"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
