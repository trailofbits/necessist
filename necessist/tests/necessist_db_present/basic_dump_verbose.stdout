examples/basic/src/lib.rs:4:5-4:12: `n += 1;` passed
examples/basic/src/lib.rs:14:9-14:16: `n += 1;` timed-out
examples/basic/src/lib.rs:21:5-21:12: `n += 1;` failed
examples/basic/src/lib.rs:28:18-28:27: `.join("")` nonbuildable
